conda_libmamba_solver/solver.py

```diff
@@ -1031,7 +1031,7 @@ def _notify_conda_outdated(
             return
 
         # we only want to check if a newer conda is available in the channel we installed it from
-        conda_newer_str = f"{channel_name}::conda>{current_conda_prefix_rec.version}"
+        conda_newer_str = f"{channel_name}::conda>{_conda_version}"
```

Comment:
Is there a chance this is the `conda` version from a non-base environment?

Reply:
I'm guessing yes if you're using condastandalone? But either way if we are trying to ensure that conda tests pass with conda-libmamba-solver as a default we need to adopt the same method for checking as them see https://github.com/conda/conda/blob/b70b7891206a598274c7dffa811aab5f5e52f665/conda/core/solve.py#L1211

Reply:
I am mostly concerned about Windows systems, where a non-base environment can inject its conda into the activation mechanism. But well, worst case it cannot be updated.